custom_components/accuweather_custom/const.py

```diff
@@ -57,3 +57,6 @@
 }
 UPDATE_INTERVAL_OBSERVATION = timedelta(minutes=40)
 UPDATE_INTERVAL_DAILY_FORECAST = timedelta(hours=6)
+UPDATE_INTERVAL_HOURLY_FORECAST = timedelta(minutes=40)
```

Comment:
Currently, with the first API key there will be 40 requests per day, with the second one - 36 per day.
If you change `UPDATE_INTERVAL_HOURLY_FORECAST` to 1h and use the second API key for the daily forecast, the first key will be used 36 times and the second 28. There will be more requests left for possible HA restarts.

Reply:
Good hint, however for now I'll leave it as it is because I prefer to have more frequent updates of hourly forecast.